src/components/createTestCase/CreateTestCase.test.tsx

```diff
@@ -1,16 +1,27 @@
 import * as React from "react";
-import { render, screen, waitFor, fireEvent } from "@testing-library/react";
+import {
+  render,
+  screen,
+  waitFor,
+  fireEvent,
+  within,
+} from "@testing-library/react";
 import { MemoryRouter } from "react-router-dom";
 import CreateTestCase from "./CreateTestCase";
 import userEvent from "@testing-library/user-event";
-import axios from "axios";
+import axios, { AxiosError, AxiosResponse } from "axios";
 import { ApiContextProvider, ServiceConfig } from "../../api/ServiceContext";
 import TestCaseRoutes from "../routes/TestCaseRoutes";
 import TestCase from "../../models/TestCase";
 
 jest.mock("axios");
 const mockedAxios = axios as jest.Mocked<typeof axios>;
 
+// mock editor to reduce errors and warnings
+jest.mock("../editor/Editor", () => () => (
```

Comment:
Is the function nesting necessary?

Reply:
Yes I believe so...for mocking a functional component the factory function (second argument in jest.mock) would need to return a function

Reply:
Gotcha, thanks!